src/assets.rs

```diff
@@ -307,18 +314,41 @@ pub(crate) fn get_serialized_integrated_syntaxset() -> &'static [u8] {
 }
 
 pub(crate) fn get_integrated_themeset() -> ThemeSet {
-    from_binary(include_bytes!("../assets/themes.bin"))
+    from_binary(include_bytes!("../assets/themes.bin"), COMPRESS_THEMES)
+}
+
+pub(crate) fn from_binary<T: serde::de::DeserializeOwned>(v: &[u8], compressed: bool) -> T {
+    asset_from_contents(v, "n/a", compressed)
+        .expect("data integrated in binary is never faulty, but make sure `compressed` is in sync!")
```

Comment:
I was shortly thinking about this before reading this piece of code. Maybe we could use the asset metadata file to store whether or not the binaries are compressed? This way, we could even make a runtime decision.

Reply:
That's certainly a tempting feature to introduce at some point!